public/editor/stylesheets/projects-style.less

```diff
@@ -33,189 +12,181 @@ h2 {
-  transform: rotate(-4deg) translateY(-1px);
-  -webkit-transform: rotate(-4deg) translateY(-1px);
-}
+  .project-ui {
+    flex-shrink: 1;
+    white-space: nowrap;
+    text-align: right;
+  }
 
-.project-favorite {
-  width: 35px;
-}
+  .published-link  {
+    text-decoration: none;
+    color: #3186c1;
+    display: inline-block;
 
-.project-favorite {
-  padding: 0;
-  text-align: left;
-}
+    &:hover {
+      text-decoration: underline;
+    }
 
-.project-favorite-button {
-  width: 30px;
-  height: 30px;
-  position: relative;
-  top: -3px;
-  background-image: url(/img/icon/star-off.svg);
-  background-repeat: no-repeat;
-  background-position: center center;
-  background-size: 20px;
-  opacity: .6;
-  display: inline-block;
+    @media screen and (max-width: 800px) {
+      display: block;
+    }
+  }
 
-  &:hover {
-    opacity: 1;
+  .project-information {
+    font-size: 0.8em;
+    color: rgba(0,0,0,.4);
+    font-style: italic;
+    white-space: nowrap;
   }
-}
 
-.project-favorite-selected {
-  background-image: url(/img/icon/star-on.svg);
-  opacity: 1;
-}
+  .project-description {
+    font-size: 0.9em;
+    color: rgba(0,0,0,.6);
+    margin-top: 5px;
+    text-overflow: ellipsis;
+    overflow: hidden;
+    white-space: nowrap;
+
+    &:empty {
+      display: none;
+    }
+  }
 
-td.project-remix {
-  text-align: right;
-  color: #A1A1A1;
-  width: 1%;
-  white-space: nowrap;
-}
+  .project-delete {
+    color: black;
+    margin-left: 12px;
+    opacity: .3;
+    cursor: pointer;
+    font-weight: 600;
+    display: inline-block;
+  }
 
-.project-button {
-  background-color: #2A9961;
-  display: inline-block;
-  padding: 8px 15px 6px 15px;
-  cursor: pointer;
-  color: white;
-  -webkit-user-select: none;
-  -moz-user-select: none;
-  -ms-user-select: none;
-  border-bottom: solid 2px rgba(0,0,0,.3);
-  border-radius: 2px;
-  position: absolute;
-  top: 0;
-  right: 0;
-}
+  .icon-garbage-can {
+    width: 15px;
+    height: 18px;
+    position: relative;
+    top: 3px;
+    display: inline-block;
+    background-image: url(/img/icon/garbage-can-bottom.svg);
+    background-repeat: no-repeat;
+    background-size: 15px;
+    background-position: bottom;
+    transform: scale(.95);
+
+    &:after {
+      content: "";
+      width: 15px;
+      height: 6px;
+      position: absolute;
+      top: 0;
+      left: 0;
+      background-image: url(/img/icon/garbage-can-top.svg);
+      background-repeat: no-repeat;
+      background-size: 17px;
+      background-position: top;
+      transition: transform .1s ease-out;
+    }
+  }
 
-.project-button:hover {
-  background-color: #39AA6D;
-}
+  .project-delete:hover {
+    opacity: .6;
+  }
 
-.project-button:active {
-  background-color: #288952;
-  top: 2px;
-  border: none;
-}
+  .project-delete:hover :after {
+    transform-origin: left;
+    -webkit-transform-origin: left;
+    transform: rotate(-4deg) translateY(-1px);
+    -webkit-transform: rotate(-4deg) translateY(-1px);
```

Comment:
Yeah totally, in fact I found a mixin that we already use in LESS to apply all the prefixes so I'll try to use that.